packages/zero-client/src/client/keys.test.ts

```diff
@@ -29,21 +29,27 @@ test('toPrimaryKeyString', () => {
       issueID: 'issue1',
       labelID: 'label1',
     }),
-  ).toMatchInlineSnapshot(`"e/issue_label/dqncq7kzfa6fpmhj26azwoy9t"`);
+  ).toMatchInlineSnapshot(
+    `"e/issue_label/15328927053344014787296837153592575662"`,
```

Comment:
What was the motivation to not use `toString(36)` any more?

Reply:
Whoops, it wasn't a conscious choice. Now I get to redo all these tests 😂.